src/main/java/Duke.java

```diff
@@ -1,10 +1,198 @@
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+
 public class Duke {
+
+    static int idx = 0;
+    static ArrayList<Task> tasks = new ArrayList<>();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        greetUser();
+
+        loadData();
+
+        serveUser();
+
+        goodbyeUser();
+
+    }
+
+    private static void loadData() {
```

Comment:
Try to extract out certain parts of the method into another method as the method has more than 30 lines of code.